tests/integration/push.rs

```diff
@@ -76,7 +76,7 @@ fn test_push_top() {
 #[test]
 fn test_push_submodule() {
     let temp_dir = crate::fixtures::toprepo::readme_example_tempdir();
-    let temp_dir = &temp_dir.into_path();
+    let temp_dir = &temp_dir.keep();
```

Comment:
Using `into_path` must be an error. 
```suggestion
    let temp_dir = temp_dir.path();
```

Reply:
Good. I updated it :)